Package.swift

```diff
@@ -4,6 +4,6 @@ let package = Package(
     name: "FluentSQLite",
     dependencies: [
         .Package(url: "https://github.com/vapor/sqlite.git", majorVersion: 1, minor: 0),
-        .Package(url: "https://github.com/vapor/fluent.git", majorVersion: 1, minor: 0),
+        .Package(url: "https://github.com/vapor/fluent.git", majorVersion: 1, minor: 3),
```

Comment:
This should be changed to `majorVersion: 1` only for both. All internal dependencies should only specify major version. Looks like we missed this when we released the 1.0 tag.